about/policy/govt-urls.md

```diff
@@ -27,32 +27,32 @@ Federal agencies [are required](https://www.whitehouse.gov/sites/whitehouse.gov/
 
 State or local agencies can browse the list by [by level of government](https://github.com/GSA/govt-urls/), you can sort by state by downloading the .csv file to your computer. Please [email updates or additions to the Search team](mailto:search@support.digitalgov.gov), or [open an issue in GitHub](https://github.com/GSA/govt-urls/issues).
 
-## What's Included in The List?
+## What's Included in the List?
 
-* Federal, state, local, [tribal](https://www.bia.gov/tribal-leaders-directory), commonwealth, and territorial government agency websites
-* [Federal reserve banks and branches](https://www.federalreserve.gov/aboutthefed/federal-reserve-system.htm)
-* [Federal home loan banks](http://www.fhlbanks.com/)
-* Libraries, archives, and museums, including [Presidential libraries](https://www.archives.gov/presidential-libraries/)
-* Department of Defense websites for [recruiting](https://www.defense.gov/Resources/Military-Departments/DOD-Websites/category/Recruiting/) and [service academies](https://www.defense.gov/Resources/Military-Departments/DOD-Websites/category/Academy/)
-* [Travel and tourism](https://www.usa.gov/state-travel-and-tourism) websites for states and U.S. territories
-* [State lotteries](https://www.usa.gov/state-lotteries)
-* [Cooperative extensions](https://nifa.usda.gov/land-grant-colleges-and-universities-partner-website-directory?state=All&type=Extension)
-* [Combined federal campaigns](https://www.opm.gov/combined-federal-campaign/find-local-campaigns/#url=CFC-Zones)
-* [Government sponsored enterprises](https://en.wikipedia.org/wiki/Government-sponsored_enterprise#List_of_GSEs) (such as Fannie Mae)
-* Federal and state retirement systems
-* Task forces (such as the Preventive Services Task Force) and commissions (such as the 9/11 Commission)
-* A few select, non-goverment organizations (such as the Red Cross) and public-private partnerships
+* Federal, state, local, [tribal](https://www.bia.gov/tribal-leaders-directory), commonwealth, and territorial government agency websites.
```

Comment:
Just for my knowledge, do all bullets need periods even if they're not complete sentences?

Reply:
Yes, alas, that's AP style.